src/plugins/yettelbank-rs/__tests__/parsers/accounts.test.ts

```diff
@@ -1,22 +1,92 @@
 import { parseAccounts } from '../../fetchApi'
-import { loadHtmlFile } from '../../helpers'
-
-const accountsBody = loadHtmlFile('__tests__/parsers/accounts.html')
```

Comment:
please use accounts.html instead of hard-coding the example html into string in this file